src/components/Accordion/AccordionItem/AccordionItem.jsx

```diff
@@ -7,29 +7,55 @@ import {AccordionContext} from '~/components/Accordion/Accordion.context';
 
 export const AccordionItem = ({id, label, icon, onClickToOpen, onClick, onClickToClose, children, ...props}) => {
     const context = useContext(AccordionContext);
-    const open = context.currentItem === id;
+    let open;
+
+    if (context.isMultipleOpenable) {
+        open = context.currentItem ? context.currentItem.includes(id) : false;
+    } else {
+        open = context.currentItem === id;
+    }
 
     const handleClick = e => {
         if (!open) {
             onClickToOpen(e);
```

Comment:
Do you think we really need all these callbacks ? onClickToOpen / onClickToClose / onClick, in addition to the setOpenedItem of Accordion ?

Reply:
Maybe not, we just keep `onClick`? Or I remove all callbacks?

Reply:
When we use the UncontrolledAccordion we haven't access to `setOpenedItem`

Reply:
We just have `onClick` with `open` as second parameters